App/src/pages/CourseBrowser.tsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Select from "react-select";
+import { Box, Button, Container, FormLabel, Heading, Stack } from "@chakra-ui/react";
+
+interface SubjectItem {
+    value: string;
+    label: string;
+}
+
+const SUBJECT_ITEMS: Array<SubjectItem> = [
+    { value: "COMP", label: "COMP" },
+    { value: "SOEN", label: "SOEN" },
+    { value: "ENGR", label: "ENGR" },
+];
+
+interface CatalogItem {
+    value: string;
+    label: string;
+}
+
+const CATALOG_ITEMS: Array<CatalogItem> = [
+    { value: "201", label: "201" },
+    { value: "301", label: "301" },
+    { value: "401", label: "401" },
+];
+
+const CourseSelector1 = () => <Select options={SUBJECT_ITEMS} />;
+const CourseSelector2 = () => <Select options={CATALOG_ITEMS} />;
+
+function App() {
```

Comment:
We shouldn't be naming functions `App` since by convention `App` is reserved for a specific component (check `App.tsx`). A good function name would be the same as the file name so in your case it will be `CourseBrowser`

Reply:
I corrected this as well, thanks.